hail/python/hail/docs/tutorials/04-aggregation.ipynb

```diff
@@ -186,12 +187,41 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "slideshow": {
+     "slide_type": "-"
```

Comment:
can you disable `rise` while working on these? This is fine here but with a large change can get annoying to cull signal from noise in the json.